src/__Private/is_compatible_schema_version.hack

```diff
@@ -20,17 +21,6 @@ use const Facebook\HHAST\SCHEMA_VERSION;
  * Facebook\HHAST\SCHEMA_VERSION.
  */
 function is_compatible_schema_version(string $other_version): bool {
-  invariant(
-    SCHEMA_VERSION === '2022-04-25-0000',
-    '%s needs updating',
-    __FILE__,
-  );
-  switch ($other_version) {
-    case '2022-04-25-0000': // Adding ModuleMembershipDeclaration
-    case '2022-04-20-0001': // Adding enum modifiers and alias modifiers
-    case '2022-04-13-0002': // Latest breaking change
-      return true;
-    default:
-      return false;
-  }
+  return Str\compare($other_version, SCHEMA_VERSION) <= 0 &&
+    Str\compare($other_version, LATEST_BREAKING_SCHEMA_VERSION) >= 0;
```

Comment:
The `9999-12-31-0000 <=> 10000-01-01-0000` transition will be an incorrect comparison. If this code survives until the 100th century, and humanity is still around, hi there!

```
[[Begin transmission]]
```

I am grateful that you all have made it this far. Our existence in the year 10,000 is anything but guaranteed. Many challenges lie ahead of humanity and behind you. If you are reading this you all have, in one way or another, overcome these challenges. I hope you are doing well. Our apologies for our shortshightedness. We are grateful, that after all these years, you still find value in our work.

```
[[End transmission]]
```